src/components/modals/Modal.vue

```diff
@@ -77,6 +83,32 @@ export default {
       color: red;
     }
   }
+  .maximize-toggle {
+    position: absolute;
+    right: 55px;
+    top: 20px;
+    cursor: pointer;
+    &:hover {
+      color: red;
+    }
+  }
+}
+
+.maximized {
+  background: rgba(0, 0, 0, 0.95);
+  padding: 0;
+  height: 100%;
+  width: 100%;
+  max-width: 100%;
+  max-height: 100%;
+}
+.not-maximized {
```

Comment:
The default styling should not change (and should not need a separate CSS class)

Reply:
Hmm, I did it this way because I wasn't sure how to revert back to the "default", in css afaik revert picks the browers default. Maybe removing the v-bind:class is enough...

Reply:
The problem here is: I can't seem to override attributes set within .characters.modal (etc) from a v-bind applied to the modal div. I _can_ override attributes set within .modal, but max-width requires different defaults for different modals so that isn't enough. I'm no expert on css specificity but the only working solution I've found is to have a .not-maxized class that the v-bind removes when it binds .maximize.

So, I've minimized the amount of code in .not-maximized, but I can't see a way to remove it entirely.